utilities/transactions/write_prepared_txn_db.h

```diff
@@ -222,7 +222,6 @@ class WritePreparedTxnDB : public PessimisticTransactionDB {
       // rare case and it is ok to pay the cost of mutex ReadLock for such old,
       // reading transactions.
       WPRecordTick(TXN_OLD_COMMIT_MAP_MUTEX_OVERHEAD);
-      ROCKS_LOG_WARN(info_log_, "old_commit_map_mutex_ overhead");
```

Comment:
I checked them and they should not be printed in normal cases or if they do they should be very rare. Let me know if you think otherwise. I am keeping them otherwise since if due to a bug/unexpected behavior they do get hit, we would have something in the log to give us a clue of why and when it happened.